server/app/controllers/budgetActions.js

```diff
@@ -25,8 +25,19 @@ const readBudgetById = async (req, res, next) => {
 
 const add = async (req, res, next) => {
   try {
-    const result = await tables.budget.create(req.body);
-    res.status(201).send(`Budget ajouté avec succès. ID : ${result.insertId}`);
+    const isCategory = await tables.category.readCategoryById({
```

Comment:
**question**: What do you mean by "isCategory" ?

Reply:
I mean does the category belongs to the user. I renamed it for more clarity "categoryBelongsToUser"